### PR TITLE
chore: Remove unused .travis.yml

### PR DESCRIPTION
**Pull request by nschonni**
The hook no longer appears to be used and doesn't build correctly for forked repos either

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +0,0 @@
-language: csharp
```
[the remark this answers is not in the document]
Comment (nschonni) [reply .travis.yml]: Good point, I'll see about inlining the content there as-is. Currently this just causes people forking the repo to get build failure emails when they create branches or PRs

[the remark this answers is not in the document]
Comment (nschonni) [reply .travis.yml]: It's already pointing to the Travis docs, so that should be good now that they have sections on .Net Core

**Comment by mairaw** [comment]
@KathleenDollard @livarcocc @dsplaisted can you guys review this PR? The original file was specific to .NET Core but has been severely outdated. Maybe we can update the travis (and the appveyor files from the previous PR) and link to their docs for extra info?

**Comment by richlander** [comment]
If we don't have a sponsor for travis integration, we should delete the sample. I agree that it is very out of date.

Stating the obvious, we should probably ensure we have good docs for DevOps integration as a higher priority.

CI samples should not be in this repo, but in the samples (or other) repo. Right?

**Comment by BillWagner** [comment]
closing based on the above conversation.

**Comment by nschonni** [comment]
@BillWagner the above conversation was to delete if it's not going to be maintained. Are you going to redo this PR to remove the file?

**Comment by mairaw** [comment]
I think so @BillWagner. I think it would also be good to sync with @erickson-doug's team on the DevOps docs related to .NET scenarios.

**Comment by mairaw** [comment]
Resolved merge conflicts and did a small fix. I'll merge when build is green.

**Review by mairaw — APPROVED** [review]

**Pull request merged**